lib/amazon_product_advertising_client/item_lookup.ex

```diff
@@ -0,0 +1,15 @@
+defmodule AmazonProductAdvertisingClient.ItemLookup do
+  @moduledoc false
+
+  alias __MODULE__
+  alias AmazonProductAdvertisingClient.Config
+  defstruct "Availability": "Available",
```

Comment:
For completeness sake, please add all of the [ItemLookup request parameters](http://docs.aws.amazon.com/AWSECommerceService/latest/DG/ItemLookup.html#ItemLookup-rp) to the struct.